src/API/OnboardingTasks.php

```diff
@@ -185,6 +185,7 @@ public static function sanitize_special_column_name_regex( $value ) {
 	 * @return string Block content.
 	 */
 	private static function get_homepage_cover_block( $image ) {
+		$shop_url = site_url( '/?post_type=product' );
```

Comment:
What do you think about using core's helper methods to retrieve this?

E.g.: `$shop_url = get_permalink( woocommerce_get_page_id( 'shop' ) );`

_See: https://www.skyverge.com/blog/get-woocommerce-page-urls/_

Reply:
Sounds much better!